test/e2e/warnings_test.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package e2e
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+	"testing"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func TestWarningsFlag(t *testing.T) {
+	minorVersion, _ := getServerMinorVersion()
```

Comment:
check that err is not nil?

Reply:
I thought of skipping the test if version was not found, but failing it makes more sense. I will update it accordingly.